qtpy/tests/test_qtwidgets.py

```diff
@@ -1,36 +1,90 @@
 """Test QtWidgets."""
+
+import sys
+
 import pytest
 
-from qtpy import PYQT5, PYQT_VERSION, QtWidgets
+from qtpy import PYQT5, PYQT_VERSION, QtCore, QtGui, QtWidgets
+from qtpy.tests.utils import using_conda, not_using_conda
 
 
-def test_qtextedit_functions():
+@pytest.mark.skipif(sys.platform.startswith('linux') and not_using_conda(),
+                    reason="Fatal Python error: Aborted on Linux CI when not using conda")
+def test_qtextedit_functions(qtbot, pdf_writer):
     """Test functions mapping for QtWidgets.QTextEdit."""
     assert QtWidgets.QTextEdit.setTabStopWidth
     assert QtWidgets.QTextEdit.tabStopWidth
     assert QtWidgets.QTextEdit.print_
+    textedit_widget = QtWidgets.QTextEdit(None)
+    textedit_widget.setTabStopWidth(90)
+    assert textedit_widget.tabStopWidth() == 90
+    print_device, output_path = pdf_writer
+    textedit_widget.print_(print_device)
+    assert output_path.exists()
 
 
-def test_qplaintextedit_functions():
+@pytest.mark.skipif(sys.platform.startswith('linux') and not_using_conda(),
+                    reason="Fatal Python error: Aborted on Linux CI when not using conda")
+def test_qplaintextedit_functions(qtbot, pdf_writer):
     """Test functions mapping for QtWidgets.QPlainTextEdit."""
     assert QtWidgets.QPlainTextEdit.setTabStopWidth
     assert QtWidgets.QPlainTextEdit.tabStopWidth
     assert QtWidgets.QPlainTextEdit.print_
+    plaintextedit_widget = QtWidgets.QPlainTextEdit(None)
+    plaintextedit_widget.setTabStopWidth(90)
+    assert plaintextedit_widget.tabStopWidth() == 90
+    print_device, output_path = pdf_writer
+    plaintextedit_widget.print_(print_device)
+    assert output_path.exists()
 
 
 def test_qapplication_functions():
     """Test functions mapping for QtWidgets.QApplication."""
     assert QtWidgets.QApplication.exec_
 
 
-def test_qdialog_functions():
+@pytest.mark.skipif(sys.platform.startswith('linux') and not_using_conda(),
+                    reason="Fatal Python error: Aborted on Linux CI when not using conda")
+@pytest.mark.skipif(sys.platform == 'darwin' and using_conda() and
+                    sys.version_info.major == 3 and sys.version_info.minor == 6,
+                    reason="Stalls on MacOS CI when using conda and Python 3.6")
```

Comment:
```suggestion
@pytest.mark.skipif(
    sys.platform == 'darwin' and using_conda() and
    sys.version_info[:2] == (3, 6)
    reason="Stalls on MacOS CI when using conda and Python 3.6")
```

Reply:
```suggestion
@pytest.mark.skipif(
    sys.platform == 'darwin' and using_conda()
    and sys.version_info[:2] == (3, 6),
    reason="Stalls on macOS CI when using Conda and Python 3.6")
```

Fix syntax error (missing comma). Also, PEP 8 specifies that binary operators should go after the line break, and that's the style we use elsewhere on these skips, AFAIK. Also, a couple minor capitalization fixes in the reason text.